atomic_reactor/__init__.py

```diff
@@ -29,7 +37,10 @@ def set_logging(name="atomic_reactor", level=logging.DEBUG, handler=None):
         handler.setLevel(logging.DEBUG)
 
         # create formatter
-        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        # NOTE: "platform:%(arch)s" is expected to be parsed, update parser in
+        # NOTE: osbs-client if this needs updating
+        fmt = '%(asctime)s platform:%(arch)s - %(name)s - %(levelname)s - %(message)s'
```

Comment:
Yes, I think that makes sense.

Could define
```
ATOMIC_REACTOR_LOGGING_FMT = '%(asctime)s platform:%(arch)s ...'
```
in osbs/constants.py and have atomic-reactor fetch it from there carefully:

```
try:
    from osbs.constants import ATOMIC_REACTOR_LOGGING_FMT
except ImportError:
    # not available in osbs < 0.41
    ATOMIC_REACTOR_LOGGING_FMT = '%(asctime)s platform:%(arch)s ...'
```